book/introduction/content-architecture.rst

```diff
@@ -1,4 +1,83 @@
 Content Architecture
 ====================
 
-TODO: Explain how Sulu websites are normally structured
+Our content is heavily structured. As mentioned in :doc:`structure` we rely on the Symfony CMF on top of 
+PHPCR. On this page we'll get into the Sulu Content Architecture.
+
+
+Sulu Instance
+-------------
+
+At first you create a Sulu Instance. It could be seen as a **single 
+installation** or one pool of data. Its real live mapping could be a company or 
+an organization.
+
+
+Webspace
+--------
+
+On your instance you'll define webspaces. Webspaces could represents your brands and corporations.
+A landing page could be a single webspace.
+
+
+Languages
+---------
+
+Once you created your webpsaces you could define languages. Pages could
+then be translated or mapped to another language as **shadow-** or **ghost-pages**.
+
+
+Page
+----
+
+As already heard in :doc:`backend-template` pages are configured in templates. They are created in
+webspaces and represent an entry in a certain menue. This pages are structured within your
```

Comment:
`These pages`, and I would also say the pages are contained within your webspace. The webspace itself does not really influence the structure of the page.